docs/vgclog_examples.md

```diff
@@ -706,3 +707,895 @@ Analyzing the structure and elements of this example log output shows that this
+
+```xml
+<exclusive-start id="1217" timestamp="2021-02-26T11:17:36.864" intervalms="1986.124">
+  <response-info timems="0.287" idlems="0.104" threads="2" lastid="00000000006EDE00" lastname="RunDataWriter.1" />
+</exclusive-start>
+<allocation-taxation id="1218" taxation-threshold="1073741824" timestamp="2021-02-26T11:17:36.865" intervalms="1986.101" />
+<gc-start id="1219" type="global mark phase" contextid="1154" timestamp="2021-02-26T11:17:36.865">
+  <mem-info id="1220" free="1438646272" total="4294967296" percent="33">
+    <remembered-set count="3514496" freebytes="157646336" totalbytes="171704320" percent="91" regionsoverflowed="3" regionsstable="0" regionsrebuilding="133"/>
+  </mem-info>
+</gc-start>
+<gc-op id="1221" type="mark increment" timems="164.843" contextid="1154" timestamp="2021-02-26T11:17:37.030">
+  <trace-info objectcount="7715572" scancount="7665293" scanbytes="214739196" />
+  <cardclean-info objects="3962203" bytes="117924792" />
+  <finalization candidates="206" enqueued="30" />
+  <ownableSynchronizers candidates="601780" cleared="16925" />
+  <references type="soft" candidates="718240" cleared="2858" enqueued="2832" dynamicThreshold="18" maxThreshold="32" />
+  <references type="weak" candidates="2321" cleared="142" enqueued="0" />
+  <references type="phantom" candidates="8" cleared="0" enqueued="0" />
+  <stringconstants candidates="9522" cleared="0"  />
+  <object-monitors candidates="7142" cleared="7066"  />
+</gc-op>
+<gc-op id="1222" type="classunload" timems="0.704" contextid="1154" timestamp="2021-02-26T11:17:37.030">
+  <classunload-info classloadercandidates="185" classloadersunloaded="13" classesunloaded="13" anonymousclassesunloaded="0" quiescems="0.000" setupms="0.644" scanms="0.043" postms="0.016" />
+</gc-op>
+<gc-end id="1223" type="global mark phase" contextid="1154" durationms="169.521" usertimems="1244.810" systemtimems="3.000" stalltimems="27.792" timestamp="2021-02-26T11:17:37.034" activeThreads="8">
+  <mem-info id="1224" free="1438646272" total="4294967296" percent="33">
+    <pending-finalizers system="30" default="0" reference="2832" classloader="0" />
+    <remembered-set count="2241440" freebytes="162738560" totalbytes="171704320" percent="94" regionsoverflowed="3" regionsstable="127" regionsrebuilding="0"/>
+  </mem-info>
+</gc-end>
+```
+
+Comparing the status of the memory at the start and end of this final `global mark phase` increment shows that:
+
+- As expected, the final global mark phase increment does not reclaim any free memory.
+- The remembered set metastructure has been rebuilt. The count has reduced and the amount of available memory in the metastructure has increased from 91% to 94%.
```

Comment:
The remembered set metastructure has been only partially (actually better, marginally) rebuilt (since, relative to the cycle start, number of overflowed regions did not change, but number of stable regions did drop from 130 to 127)

Reply:
Again, relative to cycle start, aggregate occupancy (nubmer of cards) of RSes did not change much (stayed at 94% free). Occupancy actually slightly increased since we have 3 less overflowed/stable regions.

Reply:
Not sure what you mean by '"occupancy actually slightly increased since we have now cards is RSes for 3 more regions (that used to be stable) that we successfully rebuilt."

Do you mean that the card count has increased, which is expected because we now have 3 fewer stable regions? **I am not sure what a stable region is, and how this relates to 'rebuilding'.** something we can discuss in our meeting on Tuesday. 

FYI we are comparing this snapshot at the start of the global mark cycle:

```xml
<mem-info id="1156" free="1442840576" total="4294967296" percent="33">
    <remembered-set count="2197888" freebytes="162912768" totalbytes="171704320" percent="94" regionsoverflowed="3" regionsstable="130" regionsrebuilding="0"/>
  </mem-info>

with this snapshot taken at the end of the cycle.

<mem-info id="1224" free="1438646272" total="4294967296" percent="33">
    <pending-finalizers system="30" default="0" reference="2832" classloader="0" />
    <remembered-set count="2241440" freebytes="162738560" totalbytes="171704320" percent="94" regionsoverflowed="3" regionsstable="127" regionsrebuilding="0"/>
    ```

Reply:
Changed to:

- As expected, the final global mark phase increment does not reclaim any free memory.
- The remembered set metastructure has been marginally rebuilt. The card count has increased slightly, and the number of stable regions has dropped from 130 to 127. <!--and the amount of available memory in the metastructure has increased from 91% to 94%.-->
- The number of overflow regions remains unchanged. The final global mark phase increment did not manage to rebuild any overflow regions.

Is this correct?